scripts/Win_Disk_Check_Repair.ps1

```diff
@@ -0,0 +1,56 @@
+<#
+.SYNOPSIS
+    A PowerShell script to repair a volume using the Repair-Volume cmdlet.
+
+.DESCRIPTION
+    This script uses the Repair-Volume cmdlet to scan and/or repair a specified volume
+    based on the provided DriveLetter. By default, the script performs an online scan.
+    The user can choose to perform an offline scan or a spot fix on the volume.
+
+.PARAMETER DriveLetter
+    Specifies the drive letter of the volume to be scanned/repaired.
+
+.PARAMETER Offline
+    When specified, the script performs an offline scan on the volume.
+
+.PARAMETER SpotFix
+    When specified, the script performs a spot fix on the volume.
+
+.EXAMPLE
+    .\RepairVolume.ps1 -DriveLetter "C"
+
+    Performs an online scan of the volume with the drive letter "C"
+
+.EXAMPLE
+    .\RepairVolume.ps1 -DriveLetter "C" -Offline
+
+    Schedules an offline scan of the volume with the drive letter "C" upon next reboot
+
+.EXAMPLE
+    .\RepairVolume.ps1 -DriveLetter "C" -SpotFix
+
+    Performs a spot fix on the volume with the drive letter "C"
+#>
+
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory = $true)]
+    [string]$DriveLetter,
+
+    [switch]$Offline,
+    [switch]$SpotFix
+)
+
+# Perform the requested operation(s) on the volume
+if ($Offline) {
+    Write-Host "Performing offline scan on volume $DriveLetter`:"
```

Comment:
You should use `Write-Output` instead of `Write-Host` 99% of the time. `Write-Host` writes to the console and is meant for user interaction.

<https://www.jsnover.com/blog/2013/12/07/write-host-considered-harmful/>